src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_TITLE = "supply";
+    private static final String COMA = ",";
+    private static final String BUY_TITLE = "buy";
+    private static final String RESULT_TITLE = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(fromFileName);
+        try {
+            resultLine(writeFile(fromFileName),toFileName);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    private BufferedReader writeFile(String fromFileName) {
```

Comment:
instead of this method, you should create a method that reads from a file and return these data